### PR TITLE
MINOR: Fix sensor retrieval in stand-by task's constructor

### PR DESCRIPTION
**Pull request by cadonna**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by cadonna** [comment]
Call for review: @guozhangwang @mjsax @bbejeck @vvcephei @cpettitt-confluent @ableegoldman @abbccdda

**Review by vvcephei — APPROVED** [review]
LGTM! :)

**Review by bbejeck — APPROVED** [review]

**Comment by bbejeck** [comment]
Java 11/2.12 failed with `':core:integrationTest'`
Java 11/2.13 passed
Java 8/2.11 passed

retest this please

**Review by mjsax — APPROVED** [review]

File: streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java [hunk 1]
```diff
@@ -68,8 +69,7 @@
                 final StateDirectory stateDirectory) {
         super(id, partitions, topology, consumer, changelogReader, true, stateDirectory, config);
 
-        closeTaskSensor = metrics
-            .threadLevelSensor(Thread.currentThread().getName(), "task-closed", Sensor.RecordingLevel.INFO);
+        closeTaskSensor = ThreadMetrics.closeTaskSensor(Thread.currentThread().getName(), metrics);
```
Comment (guozhangwang) [inline streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java:72]: Could you update the title / description of the PR to make a more clarifying commit message?

**Review by guozhangwang — APPROVED** [review]

**Comment by guozhangwang** [comment]
NVM, I will update the commit message while merging :)

**Pull request merged**